### ISSUE TITLE
Allow choosing a color of blur shadow

### ISSUE THREAD

**Issue by jnareb** [norbusan/pgf-blur#3]
Ordinary PGF/TikZ shadows (generic shadow, drop shadow, circular glow shadow) allow to select a color of the shadow, see e.g. this example from the pgfmanual

```tex
\usetikzlibrary {shadows}
\begin{tikzpicture}
  \foreach \i in {1,...,8}
  \node[circle,circular glow={fill=green},fill=black,text=green!50!black]
    at (\i*45:1) {Circle \i};
\end{tikzpicture}
```

It would be nice if the blur shadow allowed to select a color, instead of current solution, which uses hardcoded black (diluted).

```tex
\colorlet{pstb@shadow@color}{white!\pgfbs@opacity!black}%
```

**Comment by schtandard** [comment]
Agreed. I would be happy to provide a PR, but I'm struggling to to understand some parts of the code. For example, I don't see the point of `pstb@shadow@color` at all (changing it does not seem to affect the output). Could I get some guidance here? [Edit: After some guidance it seems like this color is used not as a color but for the opacity calculation. May investigate further, if I have time.]

Anyway, the brute force approach is to just replace the hard coded `black` with another color (it seems like there is only one that is actually used as a color and not for opacity calculation). This seems to work fine as a hack.
```latex
% tikz, shadows.blur and etoolbox need to be loaded, @ needs to be a letter.
\colorlet{blur shadow fill}{black}
\expandafter\patchcmd\csname pgfk@/tikz/render blur shadow/.@cmd\endcsname{\pgfsetfillcolor{black}}{\pgfsetfillcolor{blur shadow fill}}{}{\error}
```
